### ISSUE TITLE
Add 'Uses namespace' property to extension pages

### ISSUE THREAD

**Issue by s7eph4n** [WikiApiary/WikiApiary#208]
It might be useful to add a 'Uses namespace' property to extension pages and have a page on Wikiapiary displaying the list. Something like https://www.mediawiki.org/wiki/Extension_default_namespaces.

**Comment by jthingelstad** [comment]
Good idea and pretty easy, but will have to be updated manually. Should this be a list of names or ID's? Or both? Maybe they should be subobjects that have an ID and a name?

Sure would be neat if an extension could advertise a dependency like that via the meta API.

**Comment by s7eph4n** [comment]
The table on mw.org lists ID, constant, name and remark, but I think the most important would be the IDs, so wiki admins can avoid them when setting up custom namespaces in their wikis. This would also be sufficient for extension developers trying to avoid collision with other extensions.

Any developer who wants to specifically target the namespace set up by another extension would have to go that extensions documentation anyway, I guess.

I don't know, what the API returns, but it should be easy to just return the wgExtensionCredits as a JSON blob. These could then contain all kinds of information. But yeah, for now it is manual input.
